app/views/about/index.html.haml

```diff
@@ -14,15 +14,24 @@
         %div
           = image_tag 'hackweek-label.png', class: 'img-responsive', alt: 'Hack Week'
 .row
-  .col-md-12
+  .col-md-7
     %p.lead
       Hack Week is the time SUSE engineers experiment, innovate &amp; learn interruption-free
       for a whole week! Across teams or alone, but always without limits.
     %p.lead
       A SUSE tradition since 2007, Hack Week has brought together hundreds of hackers around
       the globe to collaborate on open source.
+    %p.lead
+      Hackweek 17 is coming up! It starts Monday, July 9th until Friday, July 13th.
```

Comment:
If you make an Episode active as admin the date will already show up. See https://hackweek.suse.com/

![screenshot from 2018-06-22 16-02-17](https://user-images.githubusercontent.com/514785/41780662-ae1c35e2-7635-11e8-84bf-a04dd6af4580.png)